spec/spec_helper.rb

```diff
@@ -0,0 +1,66 @@
+SRC_PATH = File.expand_path("../../src", __FILE__)
+DATA_PATH = File.join(File.expand_path(File.dirname(__FILE__)), "data")
```

Comment:
btwa s you already have data dir, I recommend to have command output captured there. It is quite easy on linux. simple you can run command and redirect its output to file like `update-alternatives --get-selections > data/get_selection.output`